tests/Mantis/Helper/GetLinkAttributesTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
This should be moved below the comment blocks (before the require_once call).
And follow coding guidelines.

Reply:
Moved.

> And follow coding guidelines.

Are there any other objections?